Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -287,6 +288,12 @@
     m_originalDocumentNodes.remove(uuid);
     m_replacedDocumentNodes.remove(uuid);
     m_replacementLocationOffsets.remove(uuid);
+
+    document->markers().filterMarkers(*sessionRange, [&] (const WebCore::DocumentMarker& marker) {
+        if (!std::holds_alternative<WebCore::DocumentMarker::TransparentContentData>(marker.data()))
```

Comment:
this check isn't needed; by definition, the type will be TransparentContent, which implies TransparentContentData